target/linux/ramips/dts/mt7628an_tplink_re200-v4.dts

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7628an_tplink_re200.dtsi"
+
+/ {
+	compatible = "tplink,re200-v4", "mediatek,mt7628an-soc";
+	model = "TP-Link RE200 v4";
+
+	aliases {
```

Comment:
I've recently removed the model: prefix from LED labels and therefore moved aliases, leds and state-default block to re200.dtsi. So, you can just drop it from this file as well, which will then only keep compatible and model ...